src/Microsoft.Crank.Agent/Startup.cs

```diff
@@ -4297,9 +4311,9 @@ private static async Task<string> GetAspNetRuntimeVersion(string buildToolsPath,
         /// </summary>
         private static async Task<(string Runtime, string Desktop, string AspNet, string Sdk)> GetCurrentVersions(string targetFramework)
         {
-            // There are currently no release for net8.0
+            // There are currently no release for net9.0
             // Remove once there is at least a preview and a "release-metadata" file
```

Comment:
😆 we never removed it 8.0, does this actually do anything then?

Reply:
Yes, right no we run all the benchmarks with the latest versions of 8.0, but if we force the "current" channel (`--application.channel current`) then it would for the latest public release (preview-7 right now).